core/templates/dev/head/editor/feedback_tab/FeedbackTab.js

```diff
@@ -96,19 +96,25 @@ oppia.controller('FeedbackTab', [
       backdrop: true,
       size: 'lg',
       resolve: {
-        suggestion: function() {
-          return $scope.activeThread.suggestion;
+        activeThread: function() {
+          return $scope.activeThread;
         }
       },
       controller: [
-        '$scope', '$modalInstance', 'suggestion',
-        function($scope, $modalInstance, suggestion) {
-        $scope.oldContent = suggestion.state_content.old_content;
-        $scope.newContent = suggestion.state_content.new_content;
+        '$scope', '$modalInstance', 'activeThread', 'explorationData',
+        function($scope, $modalInstance, activeThread, explorationData) {
+        var suggestion = activeThread.suggestion;
+        var states = explorationData.data.states;
+        $scope.oldContent = states[suggestion.state_name].content[0].value;
+        $scope.newContent = suggestion.state_content.value;
+        $scope.commitMessage = {
```

Comment:
The 'value' thing is legacy, I believe. I don't think we need to replicate it here, a string should do. (Unless you were thinking of adding more data to $scope.commitMessage in the future?)